modules/chatgpt.py

```diff
@@ -0,0 +1,149 @@
+from api.gooseai import GooseAI
```

Comment:
is this unused import here for some reason?

Reply:
Oh yeah that's left over from the gpt chat module I copied